Automaton/Utilities/Coords.cs

```diff
@@ -30,8 +32,8 @@ public static uint GetNearestAetheryte(int zoneID, Vector3 pos)
                 }
                 var AethersX = ConvertMapMarkerToMapCoordinate(mapMarker.X, 100);
                 var AethersY = ConvertMapMarkerToMapCoordinate(mapMarker.Y, 100);
-                var temp_distance = Math.Pow(AethersX - pos.X, 2) + Math.Pow(AethersY - pos.Z, 2);
-                if (aetheryte == default || temp_distance < distance)
+                var temp_distance = Vector3.Distance(new Vector3(AethersX, 0, AethersY), pos);
```

Comment:
why?

Reply:
I did this to standardize distances. Within the context of this function alone, it doesn't matter. But when printing and debugging directly-flying-to-next-fate distance vs teleport-to-aetheryte-then-fly distance, it helps to have comparable units of distance.